docs/src/components/banner/TableOfContentsBanner.tsx

```diff
@@ -1,57 +1,59 @@
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
 import Link from 'docs/src/modules/components/Link';
-import ROUTES from 'docs/src/route';
 import FEATURE_TOGGLE from 'docs/src/featureToggle';
 
 export default function TableOfContentsBanner() {
   return FEATURE_TOGGLE.enable_toc_banner ? (
     <Link
-      href={ROUTES.home} // Fix me!
+      href="https://ukraine.ua/news/stand-with-ukraine/" // Fix me!
       target="_blank"
       sx={(theme) => ({
         mb: 2,
-        p: 1,
         display: 'flex',
         flexDirection: 'column',
-        alignItems: 'start',
-        background:
-          theme.palette.mode === 'dark'
-            ? `linear-gradient(90deg, ${theme.palette.primary[900]}, ${theme.palette.primary[600]} 120%)`
-            : `linear-gradient(-90deg, ${theme.palette.primary[700]}, ${theme.palette.primary[500]} 120%)`,
+        overflow: 'auto',
+        border: '1px solid',
+        borderColor:
+          theme.palette.mode === 'dark' ? theme.palette.primaryDark[700] : theme.palette.grey[200],
         borderRadius: 1,
         transitionProperty: 'all',
         transitionTiming: 'cubic-bezier(0.4, 0, 0.2, 1)',
-        transitionDuration: '200ms',
+        transitionDuration: '150ms',
         '&:hover, &:focus-visible': {
-          boxShadow:
+          borderColor:
             theme.palette.mode === 'dark'
-              ? '1px 1px 20px 0 rgb(2 2 2 / 50%)'
-              : '1px 1px 20px 0 rgb(90 105 120 / 30%)',
+              ? theme.palette.primaryDark[500]
+              : theme.palette.grey[300],
         },
       })}
     >
-      <Typography component="span" variant="body1" fontWeight="bold" sx={{ color: '#fff' }}>
-        Fix me!
-        <br />
-      </Typography>
-      <Typography component="span" variant="body2" fontWeight="medium" sx={{ color: 'primary.50' }}>
-        Fix me!
+      <Typography
+        component="span"
+        variant="caption"
+        fontWeight="500"
+        color="text.primary"
+        sx={(theme) => ({
+          p: 1,
+          backgroundColor:
+            theme.palette.mode === 'dark' ? 'rgba(0, 87, 183, 0.2)' : 'rgba(0, 87, 183, 0.1)',
+        })}
+      >
+        MUI stands in solidarity with the Ukrainian people against the Russian invasion.
         <br />
       </Typography>
       <Typography
         component="span"
         variant="caption"
         fontWeight="normal"
-        sx={{
-          mt: 1,
-          pt: 1,
-          color: 'primary.50',
-          borderTop: 1,
-          borderColor: 'primary.400',
-        }}
+        color="text.primary"
+        sx={(theme) => ({
+          p: 1,
+          backgroundColor:
+            theme.palette.mode === 'dark' ? 'rgba(255, 215, 0, 0.15)' : 'rgba(255, 215, 0, 0.08)',
+        })}
       >
-        Fix me!
+        Click here to help by requesting the end of the war.
```

Comment:
Here's what I would suggest for the CTA:

```suggestion
        Find out how you can help.
```